drms/tests/test_to_datetime.py

```diff
@@ -52,6 +58,20 @@ def test_force_string(time_string, expected):
     assert drms.to_datetime(time_string, force=True) == expected
 
 
-# test_xx_list
-# test_xx_ndarray
-# test_xx_pandas_series
+@pytest.mark.parametrize('time_series, expected', [
+    (data_tai_in, data_tai_out),
+    (pd.Series(data_tai_in), data_tai_out),
+    (tuple(data_tai_in), data_tai_out),
+    (np.array(data_tai_in), np.array(data_tai_out)),
+    (dict(zip(tuple(range(len(data_tai_in))), tuple(data_tai_in))),
+     dict(zip(tuple(range(len(data_tai_out))), tuple(data_tai_out))))
```

Comment:
First of all: You forgot the `assert` keyword in the function body of `test_time_series()`. If you add the `assert` statement, you will see that the last two tests are failing.

You should always use `data_tai_out` as the expected value, because `drms.to_datetime()` always returns `pandas.Series` instances for non-scalar values. So instead of
```python
(np.array(data_tai_in), np.array(data_tai_out)),
```
you should write
```python
(np.array(data_tai_in), data_tai_out),
```
This test item should then succeed.

I don't think that we need the last test item. The fact that the `pandas.Series` constructor is called inside of `pandas.to_datetime()` is an internal implementation detail. So this might not work anymore in future implementations (and is not advertised anywhere in the documentation).

FYI:
```python
dict(enumerate(data_tai_in))
```
is equivalent to
```python
dict(zip(tuple(range(len(data_tai_in))), tuple(data_tai_in)))
```
But, as I said, better just remove this test.